daliview/build.gradle

```diff
@@ -31,22 +32,30 @@ android {
     buildTypes {
```

Comment:
Can we detect that androidABI is changed and run clean automatically? you can do from build.sh, just use ./gradlew clean